students/junghyun/users/views.py

```diff
@@ -1,3 +1,34 @@
+import json
+import re
+from sre_constants import SUCCESS
+
 from django.shortcuts import render
+from django.http  import JsonResponse
+from django.views import View
+
+from .models import User
```

Comment:
불필요한 `import`문은 지워주시고, `import`문 순서를 잘 지켜서 만들어주세요.
```suggestion
import re, json

from django.http  import JsonResponse
from django.views import View

from .models import User
```